dotnet-test-samples/schema-and-contract-testing/tests/SchemaTesting.UnitTest/CreateCustomerTests.cs

```diff
@@ -0,0 +1,46 @@
+using CreateCustomerFunction;
+
+namespace SchemaTesting.UnitTest;
+
+public class AddressTests
+{
+    [Fact]
+    public async Task ProcessCommandWithValidAddress_ShouldReturnSuccess()
+    {
+        var commandHandler = new CreateCustomerCommandHandler(options =>
```

Comment:
Consider using Automocking container to improve readability, I can explain in a call